device/iot-device-samples/send-event-x509/pom.xml

```diff
@@ -35,29 +35,4 @@
             </plugin>
         </plugins>
     </build>
-    <dependencies>
-        <!--suppress MavenDuplicateDependenciesInspection -->
-        <!--
-        This duplicate dependency declaration is only here so that it can demonstrate how to exclude
-        bouncycastle dependency from the device client library.
```

Comment:
Since we no longer have the bouncycastle dependency, we no longer need to demonstrate how to remove it